src/form-elements/FormElementCheckBoxes.tsx

```diff
@@ -65,13 +73,23 @@ function FormElementCheckboxes({
         element={element}
         required={element.required}
       >
+        {element.canToggleAll && (
+          <ToggleAllCheckbox
+            id={id}
+            element={element}
+            options={filteredOptions}
+            selected={selectedValues}
+            disabled={element.readOnly}
+            onChange={onChange}
+          />
+        )}
```

Comment:
I think this should prob be inside `<FormElementOptions options={element.options}>` as this component will render a message/loading if there are no options and this should prob not be there if there are no options, or the options are still loading.